ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test/dune

```diff
@@ -12,9 +13,10 @@
   (action (run %{dep:unixext_test.exe} -v -bt))
 )
 
-(executable
+(test
```

Comment:
@last-genius Was there a particular reason this executable was not marked as testable?

Reply:
I think I carried the example over from `ocaml/forkexecd/test/dune`, where it's marked as executable too (though it's not a cram test)

Reply:
So the binaries in forkexecd are run because there's an additional rule that depends on them when running.

For standalone test binaries, like this one, they have to be defined as tests to have them executed as part of a package's tests.